jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1634,6 +1634,9 @@ void java_bytecode_parsert::rinner_classes_attribute(
     if(!is_inner_class)
       continue;
     parsed_class.is_static_class = is_static;
+    // This is a marker that a class is anonymous.
+    if(inner_name_index == 0)
```

Comment:
Might be simpler to say `parsed_class.is_anonymous_class = inner_name_index == 0;`